src/model/server_conf.py

```diff
@@ -156,6 +156,8 @@ def create_authenticator(auth_object, temp_folder):
     else:
         raise Exception(auth_type + ' auth is not supported')
 
+    authenticator.auth_expiration_days = float(auth_object.get('expiration_days')) if auth_object.get('expiration_days') is not None else 30
```

Comment:
I think you can use default value here:
`auth_object.get('expiration_days', '30')`